lua/no-neck-pain/util/constants.lua

```diff
@@ -82,6 +82,6 @@ Co.INTEGRATIONS = {
 ---Dashboards filetypes that delays the plugin enable step until next buffer entered.
 ---
 ---@private
-Co.DASHBOARDS = { "dashboard", "alpha" }
+Co.DASHBOARDS = { "dashboard", "alpha", "ministarter" }
```

Comment:
since this requires the filetype, it should be:
```suggestion
Co.DASHBOARDS = { "dashboard", "alpha", "started" }
```

I wonder why it worked for you 🤔

Reply:
This is weird... when I try `started` instead of `ministarter` it breaks for me.
What I did was start nvim without files so I'm in the dashboard and ran `:set ft?` to see what ft was set, this returned `ministarter` so that's why I set it to that.

Here are the relevant portions of my config:
```lua

require("lazy").setup({
  {
    "echasnovski/mini.nvim",
    config = function()
      local starter = require("mini.starter")
      local nvim_header = [[ ... ]]
      starter.setup({
        header = nvim_header,
      })
  },
  -- (...)
  {
    "florzanetta/no-neck-pain.nvim",
    lazy = false,
    priority = 1000,
    version = "*",
    opts = {
      -- debug = true,
      width = 160,
      autocmds = {
        enableOnVimEnter = true,
      },
    },
  },
}
```

Reply:
ah I just realized I did a typo, it should be `starter` not `started`, and since we do a partial match on the ft, it would cover both ministarter and starter